<launch>
    <arg name="simulation_data" default="/path/to/data"/> <!-- path to the data directory -->
    <arg name="scenario_description" default="/path/to/scenario"/> <!-- path to the scenario description -->
    <arg name="simulation_rate" default="100"/> <!-- number of simulation steps per second -->

    <!-- Use the Python script with Pybind11 -->
    <node name="main_parsed_simulator_nogpu_launcher" pkg="stonefish_ros" type="gym_stonefish_nogui.py" output="screen">
        <param name="simulation_data" value="$(arg simulation_data)" />
        <param name="scenario_description" value="$(arg scenario_description)" />
        <param name="simulation_rate" value="$(arg simulation_rate)" />
    </node>
    <node name="opengym_interface" pkg="stonefish_ros" type="train_auv.py" output="screen"/>
</launch>

